### PR TITLE
fix(phone/keyboard): Swap onFocus attributes to onMouseUp to fix weapon firing

### PR DESCRIPTION
**Pull request by INeedKnockup**
**Pull Request Description**

This PR aims to fix the weapon firing bug when a user clicks to focus an element.  The onFocus attribute used for the input fields was changed to onMouseUp to prevent firing the toggleKeys function simultaneously when the user pressed their mouse click (since firing the function at the same time the click is pressed will cause the bug to re-occur given the way that SetNuiFocusKeepInput functions).

### CODE REVIEW

**Comment by itschip** [comment]
@INeedKnockup Can you please rebase this branch? :)

File: phone/src/apps/contacts/components/ContactsApp.tsx [hunk 1]
```diff
@@ -9,10 +9,11 @@ import { ContactPage } from './views/ContactsPage';
 import { ContactsThemeProvider } from '../providers/ContactsThemeProvider';
 import PersonAddIcon from '@mui/icons-material/PersonAdd';
 import Fab from '@mui/material/Fab';
-import { useHistory, useLocation } from 'react-router-dom';
+import { useHistory, useLocation } from 'react-router';
```
Comment (itschip) [inline phone/src/apps/contacts/components/ContactsApp.tsx:12]: Why do we switch to react-router here?

Comment (INeedKnockup) [reply phone/src/apps/contacts/components/ContactsApp.tsx:12]: To be honest I'm not sure how this change became part of this PR, as only my changes

Comment (itschip) [reply phone/src/apps/contacts/components/ContactsApp.tsx:12]: Please change it back :)

Comment (INeedKnockup) [reply phone/src/apps/contacts/components/ContactsApp.tsx]: This has been fixed!

**Pull request merged**